### PR TITLE
MBS-8062: query for performers and locations when rendering external event search

### PR DESCRIPTION
**Pull request by reosarevok**
Event search was not loading additional (relationship-based) info for external search, only for direct search. Changing it to do so, following the example of works.

load_ids is basically the same method as for works - not sure if we have some way of generalising this that might be good here.

### CODE REVIEW

**Comment by yvanzo** [comment]
@brainzbot retest this please.

**Comment by reosarevok** [comment]
Changed the commit message. Are these things written somewhere where I can check and get used to them? :)

**Comment by yvanzo** [comment]
Thanks. These are common practices. Some links:
- https://git-scm.com/docs/git-commit#_discussion
- https://duckduckgo.com/?q=how+to+write+commit+message

**Comment by yvanzo** [comment]
@brainzbot retest this please.

**Comment by yvanzo** [comment]
@reosarevok: Jenkins seems to get stuck on it. Can you `git rebase master` and `git push --force` it?

File: lib/MusicBrainz/Server/Data/Event.pm [hunk 1]
```diff
@@ -318,6 +318,31 @@ sub find_related_entities
     return %map;
 }
 
+=method load_ids
+
+Load internal IDs for work objects that only have GIDs.
```
Comment (mwiencek) [inline lib/MusicBrainz/Server/Data/Event.pm]: `s/work/event/` here.

Looks good otherwise except there's still some unrelated commits here.

**Comment by reosarevok** [comment]
Think it should be better now on both counts!
